sklearn/tree/tests/test_tree.py

```diff
@@ -503,6 +503,9 @@ def test_error():
 
     for name, TreeEstimator in ALL_TREES.items():
         # Invalid values for parameters
+        if name is "DecisionTreeRegressor" or name is "DecisionTreeClassifier":
```

Comment:
Add test functions or find another test function for these two classes.

Also do not compare strings with `is` (you'll get surprises because of string interning) but with `==`.